packages/theme-base/package.json

```diff
@@ -22,8 +22,5 @@
     "rimraf": "^3.0.2",
     "sass": "^1.35.2",
     "style-dictionary": "^3.0.1"
-  },
-  "dependencies": {
-    "@fontsource/inter": "^4.5.0"
```

Comment:
The bundler had issues with WOFF files inside of this dependency.

Admittedly, this is weird to me as part of a library.  Because fonts require asset files (e.g. `.woff`, `.ttf`, etc.), these have to be bundled by the client app.

Normally, I see styles only specify `font-family` and leave it to the consumer to determine **where** the fonts are located:

- https://material-ui.com/customization/typography/#self-hosted-fonts
- https://tailwindui.com/documentation#optional-add-the-inter-font-family
- https://create-react-app.dev/docs/adding-images-fonts-and-files/
- https://github.com/csstools/postcss-font-magician

Because fonts can be very expensive, I think we should revisit #165 and try to make bundle-size as much of an opt-in as possible, like https://tailwindui.com/documentation#optional-add-the-inter-font-family.

If a customer doesn't want it, `font-family` will naturally go to the next available font.

If a customer wants to load it in a blocking way, they can use `<link>`.

If a customer wants to load it in a non-blocking way, they can do what Next.js does:
> https://nextjs.org/docs/basic-features/font-optimization